r/src/main/resources/interpreter-setting.json

```diff
@@ -0,0 +1,14 @@
+[
+  {
+    "group": "knitr",
+    "name": "spark",
+    "className": "org.apache.zeppelin.rinterpreter.KnitR",
+    "properties": null
+  },
+  {
+    "group": "r",
+    "name": "spark",
```

Comment:
@jongyoul Thank you for your review. I addressed it in the latest patch. Please have a look.